src/main/java/com/minecolonies/coremod/entity/EntityFishHook.java

```diff
@@ -714,8 +714,8 @@ private ItemStack getFishingLoot(final EntityCitizen citizen)
         {
             final LootContext.Builder lootContextBuilder = new LootContext.Builder((WorldServer) this.worldObj);
             for (final ItemStack itemstack : this.worldObj.getLootTableManager()
-                                         .getLootTableFromLocation(LootTableList.GAMEPLAY_FISHING_FISH)
-                                         .generateLootForPools(this.rand, lootContextBuilder.build()))
+                                               .getLootTableFromLocation(LootTableList.GAMEPLAY_FISHING_FISH)
+                                               .generateLootForPools(this.rand, lootContextBuilder.build()))
             {
                 return itemstack;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Avoid using a branching statement as the last in a loop. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AAvoidBranchingStatementAsLastInLoop)